lib/preparser.ts

```diff
@@ -162,6 +162,7 @@ export const oclifCommandIds = [
 	'device:move',
 	'device:os-update',
 	'device:public-url',
+	`device:purge`,
```

Comment:
Not for this PR, but a thought: `oclif.manifest.json` contains this list of commands, and we already generate it during the CLI build (we should double check that it is available in all installation types, e.g. for the standalone zip, but I think it is). In this case, we could delete this command list from `preparser.ts` and use the json file - perhaps patching oclif to avoid re-loading and parsing the json file twice.

Reply:
Yeah I had the same thought.  The downside would be that of course the app architecture would not function at all without the manifest (which technically is optional), but the tradeoff is probably worth it.  Note that this is needed because of our use of spaces instead of colons (this issue crops up a lot, and has to be hacked - e.g. in help).   There's some talk about oclif supporting this in the future (which would be great, as we could remove all these hacks), but I wouldn't bet on it.